### DISCUSSION TITLE
installation issue in linux and mac

### DISCUSSION THREAD

**Discussion by FantasticForam** [EnMAP-Box/enmap-box#548] [Q&A]
Hallo Everyone.When i installed qgis and then Enmap box in my Linux pc as soon as EnMAP box is installed it shows installed in the bar without a tick on it when I try to make tick the qgis crashes and closes and shows it is due to the plugin .Moreover when i install in my macbook then i am able to install shows icon i install missing python packages and then when i close and reopen it shows python quit unexpectedly what can be done

**Comment by thielfab** [comment]
Hello, could you provide more information on your system(s), such as
* operating system + version
* QGIS version

**Comment by janzandr** [comment]
@thielfab, we are currently discussing this issue with @FantasticForam via email. I'll put you in CC next time.

**Comment by FantasticForam** [comment]
Hallo,

Linux os Ubuntu
QGIS Version 3.26.
For this QGIS Crashes after installing EnMAP box.Bcz its a old version and
I tried using Condo and mamba but it doesn’t help.
Secondly,
For personal macos13.2  where I have latest version 3.32 and EnMap box
works but the problem there is even after installing necessary Python
packages it always says its missing due to which some tasks I am unable to
do.

Best Regards,
Foram

**Comment by janzandr** [comment]
> Linux os Ubuntu
QGIS Version 3.26.
For this QGIS Crashes after installing EnMAP box.Bcz its a old version and
I tried using Condo and mamba but it doesn’t help.

Linux Ubuntu 20.04 is outdated and not longer supported by the current QGIS LongTermRelease (3.28) and latest release (3.32).
Sorry, but you have to update your Ubuntu (or change computers).

https://gis.stackexchange.com/questions/445015/installing-qgis-3-28-on-linux-mint-una-ubuntu-focal

**Comment by janzandr** [comment]
> For personal macos13.2  where I have latest version 3.32 and EnMap box
> works but the problem there is even after installing necessary Python
> packages it always says its missing due to which some tasks I am unable to
> do.

Please share screenshots, warnings and error messages.

**Comment by FantasticForam** [comment]
Yes I will just send it to you in sometime

**Comment by FantasticForam** [comment]
The below are pics for macos error.

**Reply by janzandr** [reply]
Sorry, but your pictures aren't displayed. Please repost.

**Comment by FantasticForam** [comment]
Yes the file sizes are big just a min working on it will send it again.

**Comment by FantasticForam** [comment]
OK will get it updated and then let you know if problem persits I have also
send you the pic for macos can you please let me know what problem persits
with it.
Thanku for your time and help.

Best Regards,
Foram
